web/src/app/alert/AlertContainer.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import SockJS from 'sockjs-client';
+import Stomp from 'stompjs';
+import AlertComponent from './AlertComponent';
+
+const INITIAL_RECALL_DELAY = 500;
+const RECALL_DELAY_INCREMENT = 500;
+const ALERT_LENGTH = 5000;
+const TIME_BETWEEN_ALERTS = 1000;
+
+const alertDivID = 'alertContainer';
+
+class AlertContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.stompClient = null;
```

Comment:
Probably could. I put it in the connect method since instantiating this variable is part of connecting. The connect method can probably be called from the constructor or componentDidMount